foundation_cms/campaigns/models/campaign_page.py

```diff
@@ -23,7 +26,7 @@ class CampaignPage(AbstractBasePage):
     header = models.CharField(max_length=250, blank=True, help_text="Header for the campaign page")
 
     cta = models.ForeignKey(
-        CTA,
+        Petition,
```

Comment:
Updating this to Petition for now so we can take advantage of that models additional fields such as "Show country field". If we need to update this at a later date we can do so.